packages/frames-validator/package.json

```diff
@@ -40,7 +40,7 @@
   "dependencies": {
     "@noble/curves": "^1.3.0",
     "@noble/hashes": "^1.4.0",
-    "@xmtp/node-sdk": "^0.0.27",
+    "@xmtp/node-sdk": "^0.0.29",
```

Comment:
I think you will also need to update line 3 of this file to make a new validator release

Reply:
Ah yeah, good point